### ISSUE TITLE
Definição do que será alterado do Bomberman original

### ISSUE THREAD

**Issue by cadumancini** [AdoriaStudios/Bomberbro#5]
Para a publicação do jogo, será necessário alterar pelo menos os gráficos do jogo original. Essa issue será dedicada à essa discussão.

**Comment by lorencinibrdev** [comment]
Com certeza teremos que alterar algo do gameplay. Talvez itens diferentes, mais personagens...

**Comment by cadumancini** [comment]
Sim. Também será necessário.

2015-03-09 10:55 GMT-03:00 Anderson Lorencini <notifications@github.com>:

**Comment by cadumancini** [comment]
IDEIA: Modo de batalha parecido com Tower Defense, onde tem, por exemplo, itens no centro do mapa, e os jogadores devem protegê-los de ondas de zumbis (ou outros tipos de inimigos) que nascerão nas extremidades do mapa.

**Comment by lorencinibrdev** [comment]
IDEIA: O que acham de estilizar a temática do jogo? Talvez fazer as bombas terem faces (que condizem com o que a bomba faz, por exemplo, uma cara brava apra a bomba rápida, uma cara dormindo para a bomba lenta, etc...). Seria uma boa pra fugir dos padrões.

**Comment by cadumancini** [comment]
Sim. Seguir uma temática totalmente diferente do original será estritamente
necessário para o lançamento do projeto. A ideia das faces na bomba de
acordo com o comportamento dela é bem interessante sim. Acredito que,
definindo um estilo gráfico pro jogo (por exemplo: zumbis, ou whatever),
vai dar pra pensar bem mais nesse tipo de ideias.

Em 12 de março de 2015 16:27, Anderson Lorencini <notifications@github.com>
escreveu:

**Comment by EduardoSpinelli** [comment]
Achei muito interessante a ideia de faces na bomba, esta ideia sera passada para equipe de design criar para o jogo teste, sendo assim poderemos testar esta ideia. A principio acredito que não devemos fugir da forma de jogabilidade padrão para a modalidade de game, mais sim criar ideias inovadoras para ser acrescentadas ao jogo, como itens para a customização dos personagens ou criar personagens variados de diversas temática, como exemplo um tema alienígena, este personagem por sua vez terá poderes derivadas de sua temática, havendo a possibilidade de ser criado temáticas diversificadas e uma game enorme de personagens.
